tests/e2e/specs/shopper/checkout-different-addresses.test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	uiUnblocked,
+} from '@woocommerce/e2e-utils';
+
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopper,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+
+import { BILLING_DETAILS, SHIPPING_DETAILS } from '../../../utils/constants';
+const SIMPLE_PRODUCT_NAME = '128GB USB Stick';
+
+const config = require( 'config' );
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'Skipping Checkout tests', () => {} );
+
+describe( 'Shopper → Checkout → Can have different shipping and billing addresses', () => {
+	beforeAll( async () => {
+		// prevent CartCheckoutCompatibilityNotice from appearing
+		await preventCompatibilityNotice();
+		await merchant.login();
+
+		// Activate all address fields.
+		await visitBlockPage( 'Checkout Block' );
+		await openDocumentSettingsSidebar();
+		await selectBlockByName(
+			'woocommerce/checkout-shipping-address-block'
+		);
+		await expect( page ).toClick( 'label', { text: 'Company' } );
+		await expect( page ).toClick( 'label', {
+			text: 'Apartment, suite, etc.',
+		} );
+		await expect( page ).toClick( 'label', { text: 'Phone' } );
+		await saveOrPublish();
+	} );
+
+	afterAll( async () => {
+		// Deactivate all address fields.
+		await visitBlockPage( 'Checkout Block' );
+		await openDocumentSettingsSidebar();
+		await selectBlockByName(
+			'woocommerce/checkout-shipping-address-block'
+		);
+		await expect( page ).toClick( 'label', { text: 'Company' } );
+		await expect( page ).toClick( 'label', {
+			text: 'Apartment, suite, etc.',
+		} );
+		await expect( page ).toClick( 'label', { text: 'Phone' } );
+		await saveOrPublish();
+		await merchant.logout();
+		await reactivateCompatibilityNotice();
+	} );
+
+	it( 'allows customer to have different shipping and billing addresses', async () => {
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
+		await shopper.block.goToCheckout();
+
+		await page.evaluate( () => {
```

Comment:
We can use the `unsetCheckbox` utility function here as well. So, instead of:

```js
await page.evaluate( () => {
	const checkbox = document.querySelector( '#checkbox-control-0' );
	if ( checkbox.checked ) {
		checkbox.click();
	}
} );
```

We can have:

```js
/**
 * External dependencies
 */
import { unsetCheckbox } from '@woocommerce/e2e-utils';

// Activate the "Company" field
await unsetCheckbox('#checkbox-control-0');
```

Reply:
Changed! 😀

Reply:
I think we missed this one. I can't see the change.